SECURITY.md

```diff
@@ -0,0 +1,11 @@
+# Reporting Security Issues
+
+To report a security issue, please email
+[oss-security@googlegroups.com](mailto:oss-security@googlegroups.com)
```

Comment:
who owns this group?

Reply:
Great question. I don't know. But this is the same address that Package Analysis and Scorecard are using.

Reply:
Jeff answered the question in chat.